omero/sysadmins/unix/server-linux-walkthrough.txt

```diff
@@ -60,26 +67,66 @@ the omero user.
 
 .. literalinclude:: walkthrough/step04_all_omero.sh
 
-:download:`step04_all_omero.sh <walkthrough/step04_all_omero.sh>`
 
+To Deploy OMERO.web, you can use either Nginx or Apache.
+Follow the steps to install your chosen web server.
 
 Installing Nginx
 ----------------
 
-As **root** install Nginx, copy the Nginx OMERO configuration file into the
+**The following steps are run as root.**
+
+Install Nginx, copy the Nginx OMERO configuration file into the
 Nginx configuration directory, and disable the default configuration:
 
 .. literalinclude:: walkthrough/step05_ubuntu1404_nginx.sh
 
 | :download:`step05_ubuntu1404_nginx.sh <walkthrough/step05_ubuntu1404_nginx.sh>`
-| :download:`step05_centos6_nginx.sh <walkthrough/step05_centos6_nginx.sh>`
+| :download:`step05_centos7_nginx.sh <walkthrough/step05_centos7_nginx.sh>`
+
+Installing Apache
+-----------------
+
+**The following steps are run as root.**
+
+Install Apache 2.4::
+
+	apt-get -y install apache2 libapache2-mod-wsgi
 
+As the **omero** system user, configure OMERO.web:
+	
+.. literalinclude:: walkthrough/setup_omero_apache24.sh
+
+In the Apache OMERO configuration ``apache.conf.tmp``, we explicitly set
+an alternate directory for placing sockets and lock files used by Apache modules
+using the ``WSGISocketPrefix`` directive. 
+The default location set in ``apache.conf.tmp`` needs to be modified when 
+installing OMERO on an Ubuntu/Debian system::
+
+  #see https://code.google.com/p/modwsgi/wiki/ConfigurationIssues
+  WSGISocketPrefix run/wsgi
+  # Use this on Ubuntu/Debian systems:
+  # WSGISocketPrefix /var/run/wsgi
+
+Copy the Apache OMERO configuration file into the
+Apache configuration directory, and disable the default configuration::
+
+    cp ~omero/OMERO.server/apache.conf.tmp /etc/apache2/sites-available/omero-web.conf
```

Comment:
Where is this section copied from? Seems a bit complicated and is a potential source for bugs.